scripts/upgrade

```diff
@@ -9,26 +9,20 @@
 source _common.sh
 source /usr/share/yunohost/helpers
 
-#=================================================
-# CHECK VERSION
-#=================================================
-
-upgrade_type=$(ynh_check_app_version_changed)
-
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 ynh_script_progression --message="Upgrading source files..." --weight=1
 
-ynh_setup_source --dest_dir="$install_dir" --keep="inc/digiwords.db"
+ynh_setup_source --dest_dir="$install_dir" --full_replace=1 --keep="inc/digiwords.db"
```

Comment:
We need to keep the uploaded files. I think there's something else happening that deletes the created digisteps upon upgrade.

```suggestion
ynh_setup_source --dest_dir="$install_dir" --full_replace=1 --keep="inc/digiwords.db fichiers"
```

Reply:
... or remove `--full_replace=1` for now